src/main/resources/application.yaml

```diff
@@ -81,3 +81,5 @@ logging:
   level:
     root: INFO
     ca.bc.gov.ag.efax: TRACE
+    org.springframework.web.servlet.DispatcherServlet: DEBUG
```

Comment:
can we set this level to an env var we control?

Reply:
yup. done.